backend.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   // Backend port
   backendPort: 8443,
   // Local Polkadot Kusama node
-  wsProviderUrl: 'ws://substrate-node:9944',
```

Comment:
Maybe you could tray to set this in your /etc/hosts :
127.0.0.1    localhost   substrate-node

That breaks docker compatibility

Reply:
oh yes, you are right. Shouldn't have changed this actually.